app/routes/index.js

```diff
@@ -0,0 +1,17 @@
+const choo = require('choo');
+const download = require('./download');
```

Comment:
nit: Is it weird that /app/storage.js uses `import * from blah` syntax but these other files use `require('baz')`?

Reply:
It is weird. In order to run this code on the server, for server rendering, without transpiling some of it has to stick with require style. That being the case we might want to just go with node style require everywhere.